### ISSUE TITLE
Quantity cleanup for gzip/bzip2

### ISSUE THREAD

**Issue by lushndm** [sebastianfeldmann/phpbu#156]
When I setup cleanup by quantity it's not work for target with compress. Backups are not deleted.
After turning off compress 
cleanup by quantity working good.
What can I do?

**Comment by sebastianfeldmann** [comment]
Couleur you Share your config without any passwords or other secret data.
I'm curious about the compression and encryption you are using and the cleanup configuration in particular 

I'll try to reproduce this tomorrow morning

**Comment by lushndm** [comment]
```<?xml version="1.0" encoding="UTF-8"?>
<phpbu xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:noNamespaceSchemaLocation="http://schema.phpbu.de/5.0/phpbu.xsd">
  <backups>
    <backup name="xtra">
      <!-- create a json logfile -->
      <log type="json" target="/home/appuser/mybackups/log-xtrabackup.json"/>

      <adapter type="env" name="myEnv"></adapter>

      <!-- source xtrabackup -->
      <source type="xtrabackup">
        <option name="user" value="***" />
        <option name="password" value="***" />
      </source>

      <target dirname="/backups/mysql-xtra"
              filename="xtra-%Y%m%d-%H%i"
              compress="gzip"/>

      <!-- cleanup quantity: keep the last 3 backups -->
      <cleanup type="quantity">
        <option name="amount" value="3"/>
      </cleanup>

    </backup>
  </backups>
</phpbu>
```

**Comment by lushndm** [comment]
> compress="gzip"

Same case for compress="bzip2"

**Comment by sebastianfeldmann** [comment]
what kind of source are you using?

**Comment by lushndm** [comment]
Any news?

**Comment by sebastianfeldmann** [comment]
Sorry, last days where a bit busy ;)

I found the issue and I will release a bugfix version asap.

**Comment by sebastianfeldmann** [comment]
I just released version 5.1.3 including a fix for this issue

**Comment by sebastianfeldmann** [comment]
Thanks for reporting this issue and for your patience.
